docs/dev/topology-over-raft.md

```diff
@@ -187,10 +187,10 @@ CREATE TABLE system.topology (
     topology_request text,
     transition_state text static,
     current_cdc_generation_timestamp timestamp static,
-    current_cdc_generation_uuid uuid static,
+    current_cdc_generation_uuid timeuuid static,
     unpublished_cdc_generations set<tuple<timestamp, id>> static,
```

Comment:
btw. I noticed this now, the `id` here should be `uuid` instead, right?
(or actually, it will now become `timeuuid`?)

Reply:
And I see you actually didn't change the `topology()` schema in system_keyspace.cc
(it looks like you need to change all three places: `current_cdc_generation_uuid`, `unpublished_cdc_generations`, `new_cdc_generation_data_uuid`)

btw. `cdc_generation_id_v2_type` will probably deserve some better name after this (since it no longer directly corresponds to the C++ `cdc::generation_id_v2` type which is used both for old-style generation IDs where the uuids are not timeuuids, and the new-style IDs where we're storing timeuuids there)

Reply:
I fixed all these issues. I changed the name of `cdc_generation_id_v2_type` in a new last commit.